src/app/screens/createInscription/index.tsx

```diff
@@ -0,0 +1,487 @@
+          </CardContainer>
+          <CardContainer bottomPadding>
+            <CardRow>
+              <div>{t('FEES.TITLE')}</div>
+              <div>{isLoading && <MoonLoader color="white" size={20} />}</div>
+            </CardRow>
+            {!isLoading && (
+              <>
+                <CardRow topMargin>
+                  <div>{t('FEES.INSCRIPTION')}</div>
+                  <NumericFormat
+                    value={revealServiceFee ?? 0}
+                    displayType="text"
+                    thousandSeparator
+                    suffix=" sats"
+                  />
+                </CardRow>
+                {externalServiceFee && (
+                  <CardRow topMargin>
+                    <div>{t('FEES.DEVELOPER')}</div>
+                    <NumericFormat
+                      value={externalServiceFee}
+                      displayType="text"
+                      thousandSeparator
+                      suffix=" sats"
+                    />
+                  </CardRow>
+                )}
+                <CardRow topMargin>
+                  <div>{t('FEES.TRANSACTION')}</div>
+                  <NumberWithSuffixContainer>
+                    <NumericFormat
+                      value={chainFee}
+                      displayType="text"
+                      thousandSeparator
+                      suffix=" sats"
+                    />
+                    <NumericFormat
+                      value={feeRate}
+                      displayType="text"
+                      thousandSeparator
+                      suffix=" sats/vB"
+                      renderText={(value: string) => <NumberSuffix>{value}</NumberSuffix>}
+                    />
+                  </NumberWithSuffixContainer>
+                </CardRow>
+                <CardRow topMargin>
+                  <div>{t('FEES.TOTAL')}</div>
+                  <div>
+                    <NumberWithSuffixContainer>
+                      <NumericFormat
+                        value={totalFee}
+                        displayType="text"
+                        thousandSeparator
+                        suffix=" sats"
+                      />
+                      <NumericFormat
+                        value={fiatFees}
+                        displayType="text"
+                        thousandSeparator
+                        prefix={`${currencySymbolMap[fiatCurrency]} `}
+                        suffix={` ${fiatCurrency}`}
+                        renderText={(value: string) => <NumberSuffix>{value}</NumberSuffix>}
+                      />
+                    </NumberWithSuffixContainer>
+                  </div>
+                </CardRow>
+              </>
+            )}
+          </CardContainer>
+          <Button onClick={onAdvancedSettingClick}>
+            <ButtonImage src={SettingIcon} />
+            <ButtonText>{t('EDIT_FEES')}</ButtonText>
+          </Button>
+          {showFeeSettings && (
+            <EditFee
```

Comment:
why can't we use the `TransactionSettingAlert` for the Edit fee. as it is used in other transaction screens?

Reply:
That component handles so many different scenarios like BTC payment, ordinal payment, stacks payment. I didn't want to add another method in there and confuse the logic even further. That component needs a rewrite to separate all those into separate components, so I am using a custom local edit fees component for now.